GettingStarted.md

```diff
@@ -1,12 +1,176 @@
+This Xamarin Nuget simplifies the process of integrating Localytics into Xamarin iOS, Android and Forms projects.
 
 The steps to integrate Localytics with this component mirrors the native Localytics SDK, so the following docs for the respective platforms maybe helpful:
 
 * [iOS Localytics Docs](http://docs.localytics.com/dev/ios.html)
 * [Android Localytics Docs](http://docs.localytics.com/dev/android.html)
 
+
+## Getting Started
+
+### Install the SDK
+1. Download the latest version of the Nuget package from [here](https://downloads.localytics.com/SDKs/Xamarin/Localytics-Xamarin-Latest.zip).
+2. Add a Nuget Package source in Visual Studio Preferences called localytics that points to the folder with the downloaded package.
+3. In Projects where you want to use Localytic SDK 
+4. Integrating to each of your projects
+	* Expand Packages folder in your platform Project
+	* Add a Package
+	* Select localytics as the package sources
+	* Pick the latest version of localytics
+	* Repeat the step for each platform (Android and iOS)
+	
+## Package Names
+
+* LocalyticsXamarin.iOS - API's available for using in the iOS platform from the Native SDK [iOS Localytics Docs](http://docs.localytics.com/dev/ios.html)
+* LocalyticsXamarin.Android - API's available for use on the Android platform [Android Localytics Docs](http://docs.localytics.com/dev/android.html)
+* LocalyticsXamarin.Shared - Platform specific API's that have been bridged to work from a Shared library project.
+* LocalyticsXamarin.Common - class available on Both iOS and Android using .net dependencies only. The Interfaces are the same across platforms offering similar functionality.
+
+## Xamarin Forms and Dependency Service
+LocalyticsXamarin.Common.ILocalytics interface is implemented using XamarinForms and can be available to a project using XamarinDependency Service.
+
+Add a file to each of your platform with the following contents
+
+    using System;
+    using LocalyticsXamarin.Common;
+    [assembly: Xamarin.Forms.Dependency(typeof(LocalyticsXamarin.Shared.LocalyticsXamarinForms))]
+    namespace LocalyticsXamarin.Shared
+    {
+    public class LocalyticsXamarinForms : LocalyticsSDK, ILocalytics, IPlatform
+	    {
+	    }
+    }
+
+Use the API in common modules using the interface
+```
+    ILocalytics localytics = DependencyService.Get<ILocalytics>();
+```
+
+* Non forms classes can use the same interface using
+```
+    ILocalytics localytics = LocalyticsSDK.SharedInstance;
+``` 
+
+## API Summary
+
+API's follow c# naming conventions and are translation of the API on the native platform.
+
+### Common APIs
+
+| API                    | Description  |
+|------------------------|------|
+| OpenSession  | Opens a session. Multiple calls are coallesed.  |
+| CloseSession | Open Sessions are marked with a pending close. Sessions are extended if there is localytics activity before expiry of Session timer |
```

Comment:
Is it true that it's any activity? Should really only be if there is another open right?

Pluralizing open session also makes it sound like more than can be open. I'd suggest:

Any open session is marked with a pending close.  Sessions are officially closed if the following call to open session occurs after the expiration of the session timeout.